### PR TITLE
Prepare for core GIF/PNG removal

### PR DESCRIPTION
**Pull request by NotMyFault**
@mtughan Preparation for `https://github.com/jenkinsci/jenkins/pull/5778/files`

Jenkins overrides icons in the order SVG -> PNG -> GIF, therefore modern Jenkins versions will display an SVG if available, older ones a PNG, and even older ones GIFs.

Modern Jenkins:

Older Jenkins:

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did

### CODE REVIEW

**Comment by mtughan** [comment]
Thanks @NotMyFault. I'll review this in the next few days and see about getting it merged.

**Comment by NotMyFault** [comment]
> Thanks @NotMyFault. I'll review this in the next few days and see about getting it merged.

@mtughan Any news about that one 👀 

It would be a charm to get this part released as well.

File: src/main/resources/org/jenkinsci/plugins/scriptler/ScriptlerManagement/index.jelly [hunk 1]
```diff
@@ -41,36 +41,42 @@
 							<td class="pane scriptler-toolbar" width="1%">
 								<j:choose>
 									<j:when test="${approved == null}">
-										<img width="16" height="16" title="${%fileNotAvailable}" src="${imagesURL}/16x16/red.gif" class="scriptler-icon-slot-first" />
+										<a class="scriptler-icon-slot">
+											<l:icon title="${%fileNotAvailable}" class="icon-error icon-sm" />
+										</a>
```
Comment (mtughan) [inline src/main/resources/org/jenkinsci/plugins/scriptler/ScriptlerManagement/index.jelly:46]: Please remove the `<a>` element around this icon to be the same as the approval and warning icons. Adding `class="scriptler-icon-slot"` here will result in a `margin-left` of 8px being applied to this element, where it was specifically 0px before.

**Review by mtughan — CHANGES_REQUESTED** [review]
@NotMyFault, can you make this one small change to make sure all the icons are consistent?

**Review by mtughan — APPROVED** [review]

**Comment by NotMyFault** [comment]
Thanks for merging, will add it to my list at the core PR in a bit 👏🏻

**Comment by NotMyFault** [comment]
@mtughan Thanks for merging, can you release this please?
Thanks in advance!

**Comment by NotMyFault** [comment]
I see you published a new release to GitHub, but that does not trigger a release published to the [Artifactory](https://repo.jenkins-ci.org/releases/org/jenkins-ci/plugins/scriptler/). It's still stuck on a release from November 2021.
Depending on your setup, you either need to use the maven release plugin or publish via CD.

**Comment by mtughan** [comment]
Yeah, I need to release a new version. The new release published to GitHub was just adding a proper release note for a security fix; the version had already been previously released. I'm hoping to get another PR or two in before I perform the release, plus look into enabling CD at some point.

**Comment by NotMyFault** [comment]
The next LTS baseline selection is coming up very soon, which includes the icon removal from core. Could you please trigger a release of this plugin before, to prepare it for the removal? @mtughan

**Comment by mtughan** [comment]
> The next LTS baseline selection is coming up very soon, which includes the icon removal from core. Could you please trigger a release of this plugin before, to prepare it for the removal? @mtughan

Scriptler 3.5 is released with this fix and a few others.

**Pull request merged**